test/global_state_accessor.jl

```diff
@@ -0,0 +1,24 @@
+@testset "GlobalStateAccessor" begin
+    using Ray: GLOBAL_STATE_ACCESSOR, NODE_IP_ADDRESS, get_node_to_connect_for_driver
+
+    gcs_address = read("/tmp/ray/ray_current_cluster", String)
```

Comment:
May be worth making this a constant

Reply:
I was gonna do the same with the `session_dir` as well